buddychatcli/ParticipantDataGenerator.cs

```diff
@@ -1,13 +1,33 @@
-﻿using CommandLine;
+﻿using CommandLine;
+using System;
+using System.Collections.Generic;
 using System.IO;
-using System.Runtime.CompilerServices;
+using System.Runtime.CompilerServices;
+using System.Text;
+
+
 
 [assembly: InternalsVisibleTo("BuddyChatCLI.test")]
 namespace BuddyChatCLI
 {
+    public class Project
+    {
+        public string CustomerName;
+        public string Title;
+        public DateTime Deadline;
+    }
+
     [Verb("CreateParticipantData", HelpText = "Creates Participant Data combining the existing data so far with the new signup data from the csv files into a new csv file")]
     public class ParticipantDataGenerator
-    {
+    {
+
+        public static readonly string SignUpFileName = "signup.csv";

```

Comment:
you should be able to use Program.SignUpFileName instead of defining this variable here. Vicky created a Defaults class in her PR so if it gets completed before your PR you could use that class.